package.json

```diff
@@ -40,7 +40,7 @@
   },
   "devDependencies": {
     "bower": "1.5.2",
-    "enb": ">= 0.15.0 < 1.0.0",
+    "enb": ">= 1.2.0 < 2.0.0",
```

Comment:
I want to leave support of `enb-@0.15`.

It seems that this is a reason to create `enb-files` package.